src/components/notifications.tsx

```
@@ -63,7 +63,7 @@ function NotificationFeed() {
                     <BellIcon className="size-4" />
                     <span className="sr-only">Notifications</span>
                     {metadata?.unread_count !== 0 && (
-                        <div className="absolute right-0 top-0 size-2 rounded-full bg-primary"></div>
+                        <div className="bg-primary absolute top-0 right-0 size-2 rounded-full"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)